### PR TITLE
feat/arithmetic operators closes #6

### LINKED ISSUE #6: Generate arithmetic operator
Take info [arithmetic operator](https://github.com/manugentile/slides/tree/main/sql/3%20-%20operators/3.1%20-%20arithmetic/pdf) and generate markdown

### PR DESCRIPTION
**Pull request by scriptswhisper**
Merge **after** https://github.com/manugentile/manugentile.github.io/pull/16

### CODE REVIEW

File: courses/sql/3_operators/sql-arithmetic-operators.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
+# Types of SQL operators
+
+## Arithmetic
```
Comment (manugentile) [inline courses/sql/3_operators/sql-arithmetic-operators.md]: I'd remove the Overview code from arithmetic one

**Review by manugentile — CHANGES_REQUESTED** [review]

File: courses/sql/3_operators/sql-arithmetic-operators.md (hunk 1 above)
Comment (scriptswhisper) [reply courses/sql/3_operators/sql-arithmetic-operators.md]: Sorry, I don't get it @manugentile, what do you want to change?

Comment (manugentile) [reply courses/sql/3_operators/sql-arithmetic-operators.md]: @scriptswhisper You added the overview info, like Types of SQL operators, comparison (etc) into the arithmetic operator file, can You please keep row 5 and delete until 21 and move row 5 below # Arithmetic Operators?

Comment (scriptswhisper) [reply courses/sql/3_operators/sql-arithmetic-operators.md]: Updated, thanks for the detailed info @manugentile

**Pull request merged**